src/text-expander-element.ts

```diff
@@ -199,7 +203,7 @@ class TextExpander {
 
   onKeydown(event: KeyboardEvent) {
     if (event.key === 'Escape' && (this.menu || this.combobox)) {
```

Comment:
Suggestion - Instead of checking for the presence of `menu` or `combobox` you can delegate this to the `deactivate` method and return a boolean from it. For reference see this commit - https://github.com/github/text-expander-element/pull/19/commits/cd81b9a36b6f2c951c29ab7bff4e2e71ded18c7b

Reply:
cherry-picked it back 👍